examples/apps/table-detail-tea/src/main.rs

```diff
@@ -0,0 +1,239 @@
+use std::io::stdout;
+/// A Ratatui example that demonstrates the Elm architecture with a basic list - detail
```

Comment:
Nit: wrap at 100 chars

Reply:
https://github.com/ratatui/ratatui/actions/runs/14189010632/job/39749447988?pr=1741

???

Reply:
My apologies. It was the period that made it 101 :D
I copied the line into wc -c because it looked short, and added application manually, but without the .